pom.xml

```diff
@@ -243,6 +244,16 @@
         <artifactId>commons-logging</artifactId>
         <version>1.2</version>
       </dependency>
+      <dependency>
+        <groupId>io.micrometer</groupId>
+        <artifactId>micrometer-core</artifactId>
+        <version>${micrometer.metrics.version}</version>
+      </dependency>
+      <dependency>
```

Comment:
I'm thinking that the default metrics implementation would do nothing. If someone wants to use Prometheus (or logging for that matter), then they would drop the appropriate jar file (and configuration information) on the classpath. These implementations could then be moved to another repo (accumulo-metrics?)